proca/lib/proca/server/mtt_worker.ex

```diff
@@ -368,4 +369,14 @@ defmodule Proca.Server.MTTWorker do
     |> Email.assign(:body, html_body)
     |> Email.assign(:subject, subject)
   end
+
+  defp max_messages_per_cycle() do
+    max_messages = Application.get_env(:proca, __MODULE__)[:max_messages_per_cycle]
+
+    cond do
+      is_nil(max_messages) -> 99
```

Comment:
why would max_message be nil and default to 99? above we have 
(System.get_env("MAX_MESSAGES_PER_CYCLE", "99")), does it do the same?

In general: it's awesome to have a sane defaults, but they should be defined only ones, either put 99 in a constant DEFAULT_MAX_MESSAGE_PER_CYCLE or read the System.get_env("MAX_MESSAGES_PER_CYCLE", "99")) only once.

Reply:
I didn't clarify on the issue, but ideally, say you have 150 messages ready to send, 130 for target john doe, 30 for 30 different janes.

ideally, we try to send these 30+69 for john doe, not 99 for john and nothing to the others.

it might be a complex issue and broader, but in general, we should trying to send emails to as many different targets from as many different users as possible, rather than purely by chronological order.

I'm not sure what's the best way of achieving that, may be a topic to chat on friday evening @destag ;)

Reply:
If we change by mistake config to something like this (with a typo)

```ex
config :proca, Proca.Server.MTTWorker,
  max_messages_per_cylce: 50
```

then `Application.get_env(:proca, __MODULE__)[:max_messages_per_cycle]` will return `nil` which will work as "no limit". But I agree, there should be only one source of truth, I will try to think about more elegant way